### ISSUE TITLE
Error Message

### ISSUE THREAD

**Issue by Kyle757** [topher-au/WoWmapper#11]
Hey, I can't get it to work. When I open WoWMapper it gives me an error pop-up. I'm able to press continue and just go past it but when I press plug-ins it does the same error window over and over again. If I press details it shows me this. Not sure if it'll help.

See the end of this message for details on invoking
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.IO.FileLoadException: Could not load file or assembly 'input_360, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. Operation is not supported. (Exception from HRESULT: 0x80131515)
File name: 'input_360, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' ---> System.NotSupportedException: An attempt was made to load an assembly from a network location which would have caused the assembly to be sandboxed in previous versions of the .NET Framework. This release of the .NET Framework does not enable CAS policy by default, so this load may be dangerous. If this load is not intended to sandbox the assembly, please enable the loadFromRemoteSources switch. See http://go.microsoft.com/fwlink/?LinkId=155569 for more information.
at System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
at System.Reflection.Assembly.Load(AssemblyName assemblyRef)
at WoWmapper.PluginSelectForm.RefreshInputPlugins()
at WoWmapper.PluginSelectForm..ctor()
at WoWmapper.MainForm.buttonSelectPlugin_Click(Object sender, EventArgs e)
at System.Windows.Forms.Control.OnClick(EventArgs e)
at System.Windows.Forms.Button.OnClick(EventArgs e)
at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
at System.Windows.Forms.Control.WndProc(Message& m)
at System.Windows.Forms.ButtonBase.WndProc(Message& m)
at System.Windows.Forms.Button.WndProc(Message& m)
at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

************** Loaded Assemblies **************
mscorlib
Assembly Version: 4.0.0.0
Win32 Version: 4.6.1055.0 built by: NETFXREL2
CodeBase: file:///C:/Windows/Microsoft.NET/Framework64/v4.0.30319/mscorlib.dll

WoWmapper
Assembly Version: 0.4.3.0
Win32 Version: 0.4.3.0
CodeBase: file:///C:/Program%20Files%20(x86)/World%20of%20Warcraft/Interface/AddOns/WoWmapper.exe

System.Windows.Forms
Assembly Version: 4.0.0.0
Win32 Version: 4.6.1055.0 built by: NETFXREL2
CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll

System
Assembly Version: 4.0.0.0
Win32 Version: 4.6.1055.0 built by: NETFXREL2
CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll

System.Drawing
Assembly Version: 4.0.0.0
Win32 Version: 4.6.1055.0 built by: NETFXREL2
CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll

System.Configuration
Assembly Version: 4.0.0.0
Win32 Version: 4.6.1055.0 built by: NETFXREL2
CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll

System.Core
Assembly Version: 4.0.0.0
Win32 Version: 4.6.1055.0 built by: NETFXREL2
CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll

System.Xml
Assembly Version: 4.0.0.0
Win32 Version: 4.6.1055.0 built by: NETFXREL2
CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll

Newtonsoft.Json
Assembly Version: 7.0.0.0
Win32 Version: 7.0.1.18622
CodeBase: file:///C:/Program%20Files%20(x86)/World%20of%20Warcraft/Interface/AddOns/Newtonsoft.Json.DLL

WoWmapper_Input
Assembly Version: 1.0.0.0
Win32 Version: 1.0.0.0
CodeBase: file:///C:/Program%20Files%20(x86)/World%20of%20Warcraft/Interface/AddOns/WoWmapper_Input.DLL

WoWmapperEI
Assembly Version: 0.3.0.0
Win32 Version: 1.0.0.0
CodeBase: file:///C:/Program%20Files%20(x86)/World%20of%20Warcraft/Interface/AddOns/WoWmapperEI.DLL

System.Numerics
Assembly Version: 4.0.0.0
Win32 Version: 4.6.1055.0 built by: NETFXREL2
CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Numerics/v4.0_4.0.0.0__b77a5c561934e089/System.Numerics.dll

System.Runtime.Serialization
Assembly Version: 4.0.0.0
Win32 Version: 4.6.1055.0 built by: NETFXREL2
CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Runtime.Serialization/v4.0_4.0.0.0__b77a5c561934e089/System.Runtime.Serialization.dll

System.Xml.Linq
Assembly Version: 4.0.0.0
Win32 Version: 4.0.30319.33440 built by: FX45W81RTMREL
CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Xml.Linq/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.Linq.dll

System.Data
Assembly Version: 4.0.0.0
Win32 Version: 4.6.1055.0 built by: NETFXREL2
CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_64/System.Data/v4.0_4.0.0.0__b77a5c561934e089/System.Data.dll

************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example:

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.

![untitled](https://cloud.githubusercontent.com/assets/16599745/12197208/194e5ea6-b5d3-11e5-9317-c3cf964662bb.png)

![1](https://cloud.githubusercontent.com/assets/16599745/12197234/51962d2a-b5d3-11e5-9ff1-10e0fcd1b3e8.png)

**Comment by Kyle757** [comment]
Add-on to the original post.

I've been scrolling through the closed post trying solutions suggested to people with a similar problem.  So far I've unblocked all the plug-ins, downloaded 64x bit version of silmdx, downloaded used the repair tool but still no luck. still get the same error.

**Comment by ActionleagueWoW** [comment]
I'm having the exact same issue. I've Updated my .net, done the repairs, reset drivers for my controller. Tried both versions of Slimdx. This is the error report I'm getting When I first open the program. Thanks a bunch for any help!!! I'm really looking forward to using this program if I can get this issue resolved.

 See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.UnauthorizedAccessException: Access to the path 'C:\Program Files (x86)\Wowmapper\WoWmapper-0.4.4\WoWmapper\input_ds4.xml' is denied.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at WoWmapper.Input.SettingsFile.Save()
   at DS4Driver.WoWmapper_DS4Plugin.InitializeSettings()
   at DS4Driver.WoWmapper_DS4Plugin..ctor(Boolean LoadPlugin)
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)
   at System.Reflection.RuntimeConstructorInfo.Invoke(BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.RuntimeType.CreateInstanceImpl(BindingFlags bindingAttr, Binder binder, Object[] args, CultureInfo culture, Object[] activationAttributes, StackCrawlMark& stackMark)
   at System.Activator.CreateInstance(Type type, BindingFlags bindingAttr, Binder binder, Object[] args, CultureInfo culture, Object[] activationAttributes)
   at System.Activator.CreateInstance(Type type, Object[] args)
   at WoWmapper.MainForm.InitializePlugin(String DllFile)
   at WoWmapper.MainForm.MainForm_Load(Object sender, EventArgs e)
   at System.Windows.Forms.Form.OnLoad(EventArgs e)
   at System.Windows.Forms.Form.OnCreateControl()
   at System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
   at System.Windows.Forms.Control.CreateControl()
   at System.Windows.Forms.Control.WmShowWindow(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at WoWmapper.MainForm.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)


************** Loaded Assemblies **************
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework64/v4.0.30319/mscorlib.dll
----------------------------------------
WoWmapper
    Assembly Version: 0.4.4.0
    Win32 Version: 0.4.4.0
    CodeBase: file:///C:/Program%20Files%20(x86)/Wowmapper/WoWmapper-0.4.4/WoWmapper/WoWmapper.exe
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
Accessibility
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.79.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/Accessibility/v4.0_4.0.0.0__b03f5f7f11d50a3a/Accessibility.dll
----------------------------------------
Newtonsoft.Json
    Assembly Version: 7.0.0.0
    Win32 Version: 7.0.1.18622
    CodeBase: file:///C:/Program%20Files%20(x86)/Wowmapper/WoWmapper-0.4.4/WoWmapper/Newtonsoft.Json.DLL
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
WoWmapper_Input
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/Program%20Files%20(x86)/Wowmapper/WoWmapper-0.4.4/WoWmapper/WoWmapper_Input.DLL
----------------------------------------
input_ds4
    Assembly Version: 1.4.32.0
    Win32 Version: 1.4.32
    CodeBase: file:///C:/Program%20Files%20(x86)/Wowmapper/WoWmapper-0.4.4/WoWmapper/plugins/input_ds4.dll
----------------------------------------

************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example:

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.

**Comment by topher-au** [comment]
It is recommended that you do not store the program in the program files
folder, as windows can prevent write access for the config. Alternatively,
try running the program as administrator

**Comment by ActionleagueWoW** [comment]
So I've tried a couple other things after reading that. 

1. Downloaded and installed the wowmapper setup on to my desktop and ran as admin and I get a "This program has stopped working" error. It just crashes right away.
2. Downloaded the .Zip with all the folders and just extracted on to my desktop as well and also ran as admin. I'm still getting the same errors. 

What is the recommended download package/file and installation path for it?

Thanks a bunch!

**Comment by ActionleagueWoW** [comment]
Thanks for the quick response! I tried both of those. I reinstalled to my
desktop and the program just crashes as soon as I open it. I don't know
what I'm doing wrong. Should I download the setup and the .zip with
everything and copy the .zip in to the installation folder? What is the
preferred download package in installation path?

**Comment by Kyle757** [comment]
Alright after doing like you said and putting it in a separate folder and updating the add-on. Everything outside of game seems to be running smoothly.  I'm gonna test it in-game to make sure its working correctly after I get back from my next class but now that the initial errors aren't occurring anymore I don't think it should have any problems.

**Comment by ActionleagueWoW** [comment]
Kyle, which link did you download and where did you install it? I don't
known if I did the right one.

**Comment by Kyle757** [comment]
@ActionleagueWoW I downloaded the 64 bit version and the WoWmapper_0.4.4_Setup.exe file. I had to install them into a separate folder so i just created one on my desktop.

**Comment by ActionleagueWoW** [comment]
Okay. I re-downloaded it and installed to WoWmapper folder on my desktop. The program crashes as soon as I open it. I get the "WoWmapper has stopped working" error and only an option to close the program.. Uhg. Someone does not want me to play WoW with a controller apparently. Is it because I'm using windows 10?

**Comment by Kyle757** [comment]
@ActionleagueWoW I couldn't tell you that. I'm on windows 8, I'll leave that to somewhere who knows alot more than I do in this topic.

**Comment by themonksy** [comment]
mine is doing the same. when i start it crashes rightaway iv tryed uninstaling and trying it agean. im on windows 7 86x

**Comment by themonksy** [comment]
i dont know if this helps  but when i remove the plugins it no longer auto crashes but when i reput in the 360 or ps3 plug in to load them it auto crashes

**Comment by ActionleagueWoW** [comment]
Okay I've fixed it and it's running now. Download the setup.exe and install to your desktop. Instead of downloading the slimdz 4.0 x86, download the x64. So just switching from the x86 to the x64 solved the issue and it's running fine now!

**Comment by Kyle757** [comment]
Alright, after finally getting my computer to not say there's an error everytime I open consoleport and after disabling all my add-ons to not get in-game errors I still can't get my PS4 controller to work in-game.  When I open WoWMapper it says that a Dualshock 4 controller is plugged in and at 100% but when I open the game the buttons don't work.

**Comment by themonksy** [comment]
that fixed it for me just a shame the targeting is clumsy or it be perfect

**Comment by Ryan5460** [comment]
Okay, I have the same problem as ActionleagueWoW and so far I've done everything he's done and it's still not working for me. I have no idea what you're talking about when you explain that we should download the 'Slimdz 4.0 x64 over the x86'

**Comment by topher-au** [comment]
This issue has been closed. As version 0.5 is a complete rewrite of previous versions, this issue should be resolved. If you have further troubles with 0.5 or higher, please open another issue.
